java/test/org/openqa/selenium/grid/config/JsonConfigTest.java

```diff
@@ -32,4 +34,18 @@ void shouldUseATableAsASection() {
 
     assertThat(config.get("cheeses", "selected")).isEqualTo(Optional.of("brie"));
   }
+
+  @Test
+  void ensureCanReadListOfMaps() {
+    String raw = "{\"node\":{\"detect-drivers\":false,"
+      + "\"driver-configuration\":[{\"display-name\":\"htmlunit\","
+      + "\"stereotype\":{\"browserName\":\"htmlunit\",\"browserVersion\":\"chrome\"}}]}}";
+    Config config = new JsonConfig(new StringReader(raw));
+    List<String> expected = Arrays.asList(
+      "display-name=htmlunit",
```

Comment:
I think the values need to be JSON-encoded. **NodeOptions** will not be able to convert the `stereotype` value presented here into a **Capabilities** object.